tools/epicseg/epicseg.xml

```diff
@@ -0,0 +1,358 @@
+<tool id="epicseg_segment" name="EpiCSeg - Chromatin segmentation" version="@VERSION_STRING@">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+
+        mkdir -p '${ report.files_path }' &&
+
+        ## Acquire the count matrices
+        #for $d, $dataset in enumerate( $datasets ):
+            #for $i, $mark in enumerate( $dataset.marks ):
+                ln -s -f '${mark.reads}' '${d}_${i}.bam' &&
+                ln -s -f '${mark.reads.metadata.bam_index}' '${d}_${i}.bam.bai' &&
+            #end for
+
+            Rscript '${__tool_directory__}/epicseg.R' getcounts
+                --binsize $binsize --nthreads \${GALAXY_SLOTS:-2}
+                --regions '$regions' --target '${d}.tab'
+
+                #for $i, $mark in enumerate( $dataset.marks):
+                    --mark '${mark.label}:${d}_${i}.bam' --mapq ${mark.mapq}
+                        ${mark.pairedend} --shift ${mark.shift}
+                #end for
+
+                &&
+        #end for
+
+        ## Normalize the datasets
+        #if len( $datasets ) == 1:
+            ln -s ${d}.tab ${d}_norm.tab
```

Comment:
@wkopp $d is not available here as you closed the for loop isn't it? So I assume it will be 0?

Reply:
$d actually still exists after the loop. So the statement indeed evaluates to `ln -s 0.tab 0_norm.tab` but I agree that this is confusing. I've changed that accordingly.